charts/prometheus-node-exporter/templates/daemonset.yaml

```
@@ -267,6 +273,9 @@ spec:
         - name: {{ $mount.name }}
           hostPath:
             path: {{ $mount.hostPath }}
+            {{- with $mount.hostPath.type }}
```

In my previous comment, field `volumes.hostPath.type` was referring to PodSpec. In the values' `extraHostVolumeMounts`, `type` shall not be nested.

```suggestion
            {{- with $mount.type }}
```

---

Changed to 
```
{{- with $mount.type }}
type: {{ . }}
{{- end }}
```